content/docs/learning-guide/tutorials/entities-and-prefabs/override-a-prefab.md

```diff
@@ -202,6 +146,33 @@ For a component added as an override, you can revert the addition in the same wa
 
 For a component deleted as an override, you can revert the deletion by reverting all overrides on the selected entity. However, be aware that this would remove other overrides if they are present.
 
+## Disabling prefab overrides
+
+Prefab overrides are a new feature, and it's possible that you may want to ignore them entirely, or encounter a blocking issue with them. If you wish to disable prefab override creation and editing tools in the O3DE Editor, create a settings registry file with the following contents:
+
+```json
+{
+    "O3DE": {
+        "Autoexec": {
+            "ConsoleCommands": {
+                "ed_enableDPEInspector": false,
+                "ed_enableInspectorOverrideManagement": false
+            }
+        }
+    }
+}
+```
+
+An example of such file exists as a project-specific override in the AutomatedTesting project: [`AutomatedTesting/Registry/editorpreferences.setreg`](https://github.com/o3de/o3de/blob/development/AutomatedTesting/Registry/editorpreferences.setreg)
```

Comment:
Links to o3de into the documentation should use permalinks such as the using a development link would change over time. If the editorpreferences.setreg is renamed or deleted, then the development link is no longer valid.
```suggestion
An example of such file exists as a project-specific override in the AutomatedTesting project: [`AutomatedTesting/Registry/editorpreferences.setreg`](https://github.com/o3de/o3de/blob/c8f19bbe664a89ad92007fb7674cb8c6aa165bd9/AutomatedTesting/Registry/editorpreferences.setreg)
```